core/src/main/resources/hudson/model/Job/configure.jelly

```diff
@@ -26,50 +26,66 @@ THE SOFTWARE.
   Config page. derived class specific entries should go to configure-entries.jsp
 -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form">
-
-  <l:layout type="one-column" permission="${it.EXTENDED_READ}" title="${%Config(it.displayName)}">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form" xmlns:p="/lib/hudson/project">
+  <l:layout permission="${it.EXTENDED_READ}" title="${%Config(it.displayName)}">
     <j:set var="readOnlyMode" value="${!it.hasPermission(it.CONFIGURE)}" />
 
-    <l:js src="jsbundles/config-scrollspy.js" />
-    <l:css src="jsbundles/config-scrollspy.css" />
+    <l:header>
+      <script src="${resURL}/jsbundles/section-to-sidebar-items.js" type="text/javascript" defer="true" />
+    </l:header>
+
+    <l:side-panel>
+      <div id="tasks">
+        <div class="jenkins-app-bar">
+          <div class="jenkins-app-bar__content">
+            <h1>${%Configuration}</h1>
+          </div>
+        </div>
+      </div>
+    </l:side-panel>
 
     <l:main-panel>
-    <div class="container">
-      <div class="row">
-        <div class="jenkins-config-container">
+      <div class="behavior-loading"><l:spinner text="${%LOADING}"/></div>
 
-          <div class="behavior-loading"><l:spinner text="${%LOADING}"/></div>
+      <f:form method="post" class="jenkins-form" action="configSubmit" name="config" tableClass="config-table">
```

Comment:
the `jenkins-form` class doesn't appear to be valid on this tag and doesn't get applied

Reply:
I was looking at sources for an old version ignore this

Reply:
Weird as its working for this config page but not for the one on the folder plugin.

Reply:
It’s the makeDisabled change